pages/form-detail-dummy.md

```diff
@@ -11,16 +11,114 @@ title: Form detail dummy page
 vagovprod: false
 ---
 
-### Invalid form download
-<a
-  href="https://www.va.gov/vaforms/medical/pdf/10-10EZ-fillable.pdf"
-  download
-  data-widget-type="find-va-forms-invalid-pdf-alert"
-  data-form-number="VA0927b">Download VA Form VA0927b (PDF)</a>
+<dl>
+  <div class="vads-u-margin-bottom--4">
+    <dt class="vads-u-visibility--screen-reader">Form name:</dt>
+    <dd class="va-introtext">This is an example of a form in the Forms DB that contains a link to a PDF that has been flagged as invalid. The related forms are also invalid.</dd>
+  </div>
+  <div class="vads-u-margin-y--1">
+    <dt class="vads-u-font-weight--bold vads-u-display--inline">
+      Related to:
+    </dt><dd class="vads-u-display--inline">
```

Comment:
Not sure if we care about formatting here but this could probably be formatted a bit for readability 🙂

Reply:
This is a throwaway page that I copied out of a page source. I don't want it to appear as something we want to keep around! Lol.